src/main/osd/osd_elements.c

```diff
@@ -64,6 +64,7 @@
 
 #include "drivers/display.h"
 #include "drivers/max7456_symbols.h"
+#include "drivers/pwm_output.h"
```

Comment:
What is this one used for?

Reply:
For `getDshotTelemetry()`in
```
static int getEscRpm(int i)
{
#ifdef USE_RPM_FILTER
    if (motorConfig()->dev.useDshotTelemetry) {
        return 100.0f / (motorConfig()->motorPoleCount / 2.0f) * getDshotTelemetry(i);
    }
#endif
    if (featureIsEnabled(FEATURE_ESC_SENSOR)) {
        return calcEscRpm(getEscSensorData(i)->rpm);
    } else {
        return 0;
    }
}
```
It was already getting indirectly included through some other header so it would compile. But I figured it would be better to include it literally to prevent stumbling across problems later.